pkg/generator/appconfiguration/generator/workload/network/ports_generator.go

```diff
@@ -163,16 +163,13 @@ func (g *portsGenerator) generateK8sSvc(public bool, ports []network.Port) *v1.S
 
 		// only support Aliyun SLB for now, and set SLB spec by default.
 		svc.Annotations[aliyunLBSpec] = aliyunSLBS1Small
+		svc.Labels[kusionControl] = "true"
```

Comment:
why you need to add this label? K8s Service will Inherit the label that Collaset/Deployment have, does they don't have the label of kusionControl?

Reply:
This label is used to cooperate with the KAFE svc controller to implement gracefully rollout. The KAFE svc controller will only process svcs with this label and it is given in the svc sample before.

Reply:
I know this. But I want to know does this label is not attached to the KAFE workload, Collaset? Does it only need to attach to the service?
This label works only for KAFE system, should not get added under other situation.

Reply:
It only works with the KAFE workload. We will move this label and all Aliyun-related labels to platform.k in the next version